.travis.yml

```diff
@@ -2,16 +2,16 @@ language: android
 sudo: required
 dist: trusty
 
+before_install:
+  - mkdir "$ANDROID_HOME/licenses" || true
+  - echo -e "\n8933bad161af4178b1185d1a37fbf41ea5269c55\nd56f5187479451eabf01fb78af6dfcb131a6481e" > "$ANDROID_HOME/licenses/android-sdk-license"
+  - echo -e "\n84831b9409646a918e30573bab4c9c91346d8abd\n504667f4c0de7af1a06de9f4b1727b84351f2910" > "$ANDROID_HOME/licenses/android-sdk-preview-license"
+
 android:
   components:
-  - tools
-  - platform-tools
-  - tools
-  - build-tools-26.0.2
-  - android-26
```

Comment:
Are these removed because they're the defaults?

Reply:
No, but the Android Gradle Plugin downloads these for you (hence the licensing stuff). You can see it start here: https://travis-ci.org/bumptech/glide/jobs/310330628#L1017.